src/Microsoft.DotNet.Tools.Builder/CompileContext.cs

```diff
@@ -148,14 +148,14 @@ private static bool AnyMissingIO(ProjectContext project, IEnumerable<string> ite
                 return false;
             }
 
-            Reporter.Verbose.WriteLine($"\nProject {project.ProjectName()} will be compiled because expected {itemsType} are missing: ");
+            Reporter.Output.WriteLine($"\nProject {project.ProjectName()} will be compiled because expected {itemsType} are missing: ");
 
             foreach (var missing in missingItems)
```

Comment:
You need to tell the user just enough that something happened. To see why they'll have to use verbose.